DatadogSessionReplay/Tests/Utilities/CacheTests.swift

```diff
@@ -40,6 +40,17 @@ class CacheTests: XCTestCase {
         XCTAssertNil(cache.value(forKey: "one"))
     }
 
+    func test_bytesLimit() {
+        let cache = Cache<String, Int>(dateProvider: {
+            return Date(timeIntervalSinceReferenceDate: 0)
+        }, totalBytesLimit: 1)
+
+        cache.insert(1, forKey: "one", size: 1)
+        cache.insert(1, forKey: "two", size: 1)
+        XCTAssertNil(cache.value(forKey: "one"))
+        XCTAssertNotNil(cache.value(forKey: "two"))
```

Comment:
By design it removes oldest entry after reaching total bytes limit

Reply:
💡😃 
```suggestion
        cache.insert(1, forKey: "one", size: 1)
        cache.insert(1, forKey: "two", size: 1)
        XCTAssertNil(cache.value(forKey: "one"), "It should remove oldest entry after reaching total bytes limit")
        XCTAssertNotNil(cache.value(forKey: "two"))
```